modules/masterbar.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Module Name: WordPress.com Toolbar
+ * Module Description: An admin bar providing similar experience to the front-end navigation on WordPress.com.
+ * Sort Order: 38
+ * Recommendation Order: 16
+ * First Introduced: 4.7
+ * Requires Connection: Yes
+ * Auto Activate: No
+ * Module Tags: General
+ * Additional Search Queries: adminbar, masterbar
+ */
+
+include dirname( __FILE__ ) . '/masterbar/masterbar.php';
+
+new A8C_WPCOM_Masterbar;
```

Comment:
Thanks for suggesting this! Added domain prefetching in 0660ea26a58439a22995179e04b38be434d821f6.